components/brave_wallet/browser/zcash/zcash_scan_blocks_task.h

```diff
@@ -37,11 +41,11 @@ class ZCashScanBlocksTask {
 
   void Start();
 
+  void set_max_tasks_in_progress(uint32_t tasks) {
+    max_tasks_in_progress_ = tasks;
+  }
```

Comment:
Let's name it _for_testing if it's for testing only

Reply:
Will do in follow up